reVX/x_mission_paths/file_handlers.py

```diff
@@ -0,0 +1,271 @@
+"""
+Functions for loading substations, transmission line, etc
+
+Mike Bannister
+5/18/2021
+"""
+import os
+import logging
+
+import numpy as np
+import pandas as pd
+import geopandas as gpd
+from shapely.geometry import Point
+import rasterio as rio
+
+from .config import TEMPLATE_SHAPE, power_classes, power_to_voltage, \
+    BARRIERS_MULT
+from .utilities import RowColTransformer
+
+logger = logging.getLogger(__name__)
+
+
+class LoadData:
```

Comment:
We should move all of these layers into a single h5. Then we can just load the allconns data and the "exclusions .h5" flie using: 
https://github.com/NREL/reV/blob/e8616be663ddc590221c34f104b635252157e925/reV/handlers/exclusions.py#L18